zeppelin-zengine/src/main/java/com/nflabs/zeppelin/notebook/Note.java

```diff
@@ -231,6 +232,9 @@ public void run(String paragraphId) {
     p.setNoteReplLoader(replLoader);
     p.setListener(jobListenerFactory.getParagraphJobListener(this));
     Interpreter intp = replLoader.get(p.getRequiredReplName());
+    if (intp == null) {
```

Comment:
When user try to use non-exists interpreter, for example run `%nonexist statment`, it raise null pointer exception at previous line and this conditional block won't be executed.

In this case, there's an exception in log file
```
ERROR [2015-04-02 22:01:36,584] ({WebSocketWorker-8} NotebookServer.java[runParagraph]:431) - Exception from run
java.lang.NullPointerException
	at com.nflabs.zeppelin.notebook.NoteInterpreterLoader.get(NoteInterpreterLoader.java:70)
	at com.nflabs.zeppelin.notebook.Note.run(Note.java:234)
	at com.nflabs.zeppelin.socket.NotebookServer.runParagraph(NotebookServer.java:428)
	at com.nflabs.zeppelin.socket.NotebookServer.onMessage(NotebookServer.java:96)
	at org.java_websocket.server.WebSocketServer.onWebsocketMessage(WebSocketServer.java:469)
	at org.java_websocket.WebSocketImpl.decodeFrames(WebSocketImpl.java:368)
	at org.java_websocket.WebSocketImpl.decode(WebSocketImpl.java:157)
	at org.java_websocket.server.WebSocketServer$WebSocketWorker.run(WebSocketServer.java:657)
```

but gui does not display it.
![image](https://cloud.githubusercontent.com/assets/1540981/6964574/7b46b4a8-d985-11e4-85df-28de3a0d5a92.png)

Could you handle this case, too?